BUILD

```diff
@@ -703,6 +702,7 @@ cc_library(
     "include/grpc++/status.h",
     "include/grpc++/status_code_enum.h",
     "include/grpc++/stream.h",
+    "include/grpc++/thread_pool.h",
```

Comment:
Agree

On Wed, Jul 15, 2015 at 1:11 PM Vijay Pai <notifications@github.com> wrote:

> In BUILD <https://github.com/grpc/grpc/pull/2460#discussion_r34723063>:
>
> > @@ -703,6 +702,7 @@ cc_library(
> >      "include/grpc++/status.h",
> >      "include/grpc++/status_code_enum.h",
> >      "include/grpc++/stream.h",
> > +    "include/grpc++/thread_pool.h",
>
> I will do so, because I think it's a good idea to maintain consistency.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/grpc/grpc/pull/2460/files#r34723063>.
>